mopidy/local/storage.py

```diff
@@ -9,5 +9,6 @@
 def check_dirs_and_files(config):
     if not os.path.isdir(config['local']['media_dir']):
         logger.warning(
```

Comment:
I think this will end up being printed as:

```
Local media dir foo does not exist or we lack                     permissions to the directory or one of its parents
```

What you can do instead is:

```
        logger.warning(
            'Local media dir %s does not exist or we lack permissions to the '
            'directory or one of its parents', config['local']['media_dir'])
```

This also fixes that we should be using the full 80 chars line width, and a pre-existing mistake where we didn't let the warning call handle the string substitution like we should have.